src/main/asciidoc/repositories.adoc

```diff
@@ -1384,12 +1425,14 @@ To rather use XML to define the data the repositories shall be populated with, y
 ====
 
 [[web.legacy]]
-=== Legacy web support
+=== Legacy Web Support
```

Comment:
I wonder whether we should remove this section as we have older version documentation available at docs.spring.io.

Reply:
Very much agreed. I removed it.